src/realm/parser/generated/query_flex.cpp

```diff
@@ -101,7 +101,6 @@ typedef int16_t flex_int16_t;
 typedef uint16_t flex_uint16_t;
 typedef int32_t flex_int32_t;
 typedef uint32_t flex_uint32_t;
-typedef uint64_t flex_uint64_t;
```

Comment:
This should not be included. Lets' merge like this right now, and then I will have a look at this.